### DISCUSSION TITLE
Can we add classes to waybar elements?

### DISCUSSION THREAD

**Discussion by abraxas86** [Alexays/Waybar#3104] [Q&A] [answered]
Is there any way to add classes to waybar elements?  For example, there are a few modules I'd like to have styled in the same way.  If they could all be classed, then I could target them all at the same time in the .css file by styling by class instead of by ID.  I've looked around, but I can't find anything - I'm assuming it's not possible but I thought I'd ask before fully throwing in the towel.

**Comment by ordy** [comment, marked as answer]
No you can't, not unless you edit the source code to manually add them to your modules.

So either use the id's, css selectors or existing classes to target them.

**Reply by abraxas86** [reply]
bummer.  Thanks!  Glad I didn't overlook something, though :)
